flytekit/models/presto.py

```diff
@@ -0,0 +1,77 @@
+from __future__ import absolute_import
+
+## Todo - change this to qubole_presto once Luis's PR get's merged
+# from flyteidl.plugins import qubole_presto as _qubole
+from flyteidl.plugins import presto_pb2 as _presto
+
+from flytekit.models import common as _common
+
+
+class PrestoQuery(_common.FlyteIdlEntity):
+    def __init__(self, routing_group, catalog, schema, statement):
+        """
+        Initializes a new PrestoQuery.
+
+        :param string routing_group:
+        :param string catalog:
+        :param string schema:
+        :param string statement:
+
+        """
+        self._routing_group = routing_group
+        self._catalog = catalog
+        self._schema = schema
+        self._statement = statement
+
+    @property
+    def routing_group(self):
+        """
+        The query string.
+        :rtype: str
+        """
+        return self._routing_group
+
+    @property
+    def catalog(self):
+        """
+        :rtype: int
+        """
+        return self._catalog
+
+    @property
+    def schema(self):
+        """
+        :rtype: int
+        """
+        return self._schema
+
+    @property
+    def statement(self):
```

Comment:
Why is it called Statement here and query up there?

Reply:
I'll rename it  to `statement`. In Presto, a query is made  up  of not  just the main query "statement"  but also the  routing group, catalog,  etc